### PR TITLE
Update: Remove "Dismiss" action from Toast

### PR DESCRIPTION
**Pull request by edburyenegren-okta**
### Description

This removes the "Dismiss" action from Toast, instead relying on the transient state to handle removal.

### CODE REVIEW

File: packages/odyssey-storybook/src/components/odyssey-mui/Infobox/Infobox.mdx [hunk 1]
```diff
@@ -116,6 +116,97 @@ Infobox should always include their main messaging within the body content. Titl
...
+        <th scope="column">Component</th>
+        <th scope="column">Available Variants</th>
+        <th scope="column">
+          <span class="has-ods-tooltip">
+            <abbr aria-describedby="tip-dynamic">Dynamic</abbr>
+            <aside
+              class="ods-tooltip is-ods-tooltip-top"
+              id="tip-dynamic"
+              role="tooltip"
+            >
```
Comment (KevinGhadyani-Okta) [inline packages/odyssey-storybook/src/components/odyssey-mui/Infobox/Infobox.mdx:136]: Why these classes?

File: packages/odyssey-storybook/src/components/odyssey-mui/Toast/Toast.stories.tsx [hunk 2]
```diff
@@ -67,7 +67,7 @@ const DefaultTemplate: Story = (args) => {
       </Button>
       <Stack spacing={2} sx={{ width: "100%" }}>
         <Snackbar open={open} autoHideDuration={6000} onClose={handleClose}>
-          <Alert onClose={handleClose} severity={args.severity} variant="toast">
+          <Alert severity={args.severity} variant="toast">
```
Comment (KevinGhadyani-Okta) [inline packages/odyssey-storybook/src/components/odyssey-mui/Toast/Toast.stories.tsx:70]: What's severity?

File: packages/odyssey-storybook/src/components/odyssey-mui/Infobox/Infobox.mdx (hunk 1 above)
Comment (edburyenegren-okta) [reply packages/odyssey-storybook/src/components/odyssey-mui/Infobox/Infobox.mdx:136]: These are a holdover from the legacy docs. I copy/pasted them here for consistency.

**Review by KevinGhadyani-Okta — APPROVED** [review]
Approved, but I have a non-blocking question.

File: packages/odyssey-storybook/src/components/odyssey-mui/Infobox/Infobox.mdx (hunk 1 above)
Comment (KevinGhadyani-Okta) [reply packages/odyssey-storybook/src/components/odyssey-mui/Infobox/Infobox.mdx:136]: Could we get rid of them?

**Pull request merged**